codeHF/dpl-config_run5_hf.json

```
@@ -643,6 +687,28 @@
         },
        "processMC": "true"
     },
+    "hf-task-lb": {
+        "cutYCandMax": "2.0",
+        "d_selectionFlagLb": "0",
+        "pTBins": {
+            "values": [
+                "0",
+                "0.5",
+                "1",
+                "2",
+                "3",
+                "4",
+                "5",
+                "7",
+                "10",
+                "13",
+                "16",
+                "20",
+                "24"
+            ]
+        },
+       "processMC": "true"
```

```suggestion
        }
```
There is no such switch.